shared/src/test/scala/InterpreterTests.scala

```diff
@@ -161,4 +164,20 @@ class InterpreterTests extends AnyFunSuite:
     assertResult((VNum(6), "foo"))((ctx1.getVar("x"), ctx2.getVar("x")))
   }
 
+  test("Numeric literals are handled correctly") {
```

Comment:
We may have to find a different testing [style](https://www.scalatest.org/user_guide/selecting_a_style) so that each of these asserts can be its own test. Or we could use [`Checkpoints`](https://www.scalatest.org/scaladoc/3.2.15/org/scalatest/Checkpoints.html) for this particular test